tools/ports/freetype.py

```diff
@@ -7,21 +7,21 @@
 import shutil
 from pathlib import Path
 
-TAG = 'version_1'
-HASH = '0d0b1280ba0501ad0a23cf1daa1f86821c722218b59432734d3087a89acd22aabd5c3e5e1269700dcd41e87073046e906060f167c032eb91a3ac8c5808a02783'
+TAG = 'VER-2-11-1'
+HASH = '5e53276cf0248c61782c6410856230a287c583efdbb8959d4b1d25efdbe8c665ca45a9c952c33c710b595fcf76905375487d3803fb39c2b96dbe253c7664662c'
 
 
 def needed(settings):
   return settings.USE_FREETYPE
 
 
 def get(ports, settings, shared):
-  ports.fetch_project('freetype', 'https://github.com/emscripten-ports/FreeType/archive/' + TAG + '.zip', 'FreeType-' + TAG, sha512hash=HASH)
+  ports.fetch_project('freetype', 'https://gitlab.freedesktop.org/freetype/freetype/-/archive/' + TAG + '/freetype-' + TAG + '.zip', 'freetype-' + TAG, sha512hash=HASH)
```

Comment:
Can we use the github releases at https://github.com/freetype/freetype instead?   I think it would be good to avoid depending on another server being up (and we already depend on github.com).

Reply:
Thanks. The PR should now use GitHub.

I would prefer to use GitHub over hosting it ourselves if at all possible. We'd replace one problem (old version with unpatched vulnerabilities) with the same problem down the road (old version with unpatched vulnerabilities).

There's a link to the Freetype Gitlab on freedesktop.org on their page. From there, I found the link:  https://gitlab.freedesktop.org/freetype/freetype/-/archive/VER-2-11-1/freetype-VER-2-11-1.zip

Reply:
The file I mirrored is the office tar archive.    I believe the official signed releases are here: https://download.savannah.gnu.org/releases/freetype/.   I mirrored freetype-2.11.1.tar.gz from there.    The good thing about using our mirror is that it works even when `download.savannah.gnu.org` is down.   But if you prefer to link to  `download.savannah.gnu.org` I would be fine with that too.     I think its more stable and more official that a zip file generated from a git tag.

Reply:
> The file I mirrored is the official signed tar archive. I believe the official signed releases are here: https://download.savannah.gnu.org/releases/freetype/. I mirrored freetype-2.11.1.tar.gz from there. The good thing about using our mirror is that it works even when `download.savannah.gnu.org` is down. But if you prefer to link to `download.savannah.gnu.org` I would be fine with that too. I think its more stable and more official that a zip file generated from a git tag.

Reply:
Let's go with download.savannah.gnu.org,then. It will be easier to update when it's time to swap to a newer version.